config/releases.exs

```diff
@@ -18,7 +18,9 @@ config :bullhorn,
 
 config :appsignal, :config,
   push_api_key: bullhorn_config["APPSIGNAL_PUSH_KEY"],
-  env: bullhorn_config["APPSIGNAL_ENV"]
+  env: bullhorn_config["ENVIRONMENT"]
+
+config :bullhorn, Bullhorn.Tracer, env: bullhorn_config["ENVIRONMENT"]
```

Comment:
will changing to `ENVIRONMENT` make staging and production report under the same env?

Reply:
looked it up in our infra: `ENVIRONMENT` will be separate with `staging` and `production`. 👍